File: examples/notes/delete/node/deleteANote.js
const { NotesService } = require("m3o/notes");

// Delete a note
async function deleteAnote() {
  let notesService = new NotesService(process.env.MICRO_API_TOKEN);
  let rsp = await notesService.delete({
    id: "63c0cdf8-2121-11ec-a881-0242e36f037a",
  });
  console.log(rsp);
}

deleteAnote();

File: examples/notes/delete/go/deleteANote.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/notes"
)

// Delete a note
func DeleteAnote() {
	notesService := notes.NewNotesService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := notesService.Delete(&notes.DeleteRequest{
		Id: "63c0cdf8-2121-11ec-a881-0242e36f037a",
	})
	fmt.Println(rsp, err)
}
